public/emojify.js

```diff
@@ -1,24 +1,161 @@
+    'aquarius', 'pisces', 'id', 'phone_off', 'vibration_mode', 'eight_pointed_star', 'vs', 'white_flower', 'a', 'b',
+    'ab', 'cl', 'o', 'sos', 'cross_mark', 'circle', 'no_entry', 'name_badge', 'prohibited', '100', 'anger',
+    'hot_springs', 'no_pedestrians', 'do_not_litter', 'no_bicycles', 'non-potable_water', 'underage', 'no_phones',
+    'no_smoking', 'exclamation', 'grey_exclamation', 'question', 'grey_question', 'bangbang', 'interrobang',
+    'low_brightness', 'brightness', 'part_alternation', 'warning', 'children_crossing', 'trident', 'beginner',
+    'recycle', 'check', 'stock_market', 'sparkle', 'eight_spoked_asterisk', 'x', 'www', 'cute', 'metro', 'cyclone',
+    'zzz', 'atm', 'wc', 'accessible', 'parking', 'passport_control', 'customs', 'baggage_claim', 'locker', 'mens',
+    'womens', 'baby_change_station', 'restroom', 'put_litter_in_its_place', 'cinema', 'cell_reception', 'symbols',
+    'info', 'abc', 'abcd', 'capital_abcd', 'ng', 'squared_ok', 'squared_up', 'cool', 'new', 'free', 'ten', '1234',
+    'play', 'fast_forward', 'rewind', 'double_up', 'double_down', 'play_reverse', 'upvote', 'downvote', 'right', 'left',
+    'up', 'down', 'upper_right', 'lower_right', 'lower_left', 'upper_left', 'up_down', 'left_right', 'forward', 'reply',
+    'heading_up', 'heading_down', 'shuffle', 'repeat', 'repeat_one', 'counterclockwise', 'clockwise', 'music',
+    'musical_notes', 'plus', 'minus', 'division', 'multiplication', 'dollars', 'exchange', 'tm', 'wavy_dash', 'loop',
+    'double_loop', 'end', 'back', 'on', 'top', 'soon', 'check_mark', 'checkbox', 'radio_button', 'white_circle',
+    'black_circle', 'red_circle', 'blue_circle', 'red_triangle_up', 'red_triangle_down', 'small_orange_diamond',
+    'small_blue_diamond', 'large_orange_diamond', 'large_blue_diamond', 'black_and_white_square',
+    'white_and_black_square', 'black_small_square', 'white_small_square', 'black_medium_small_square',
+    'white_medium_small_square', 'black_medium_square', 'white_medium_square', 'black_large_square',
+    'white_large_square', 'speaker', 'mute', 'softer', 'louder', 'notifications', 'mute_notifications', 'megaphone',
+    'loudspeaker', 'umm', 'thought', 'spades', 'clubs', 'hearts', 'diamonds', 'joker', 'playing_cards', 'mahjong',
+    'time', 'checkered_flag', 'triangular_flag', 'crossed_flags'];
 
 emobox = document.getElementById("emobox");
-emojiStr = "";
-var i;
-for (i = 0; i < emojis.length; i++) {
-	emojiStr +=
-		'<li><img src="/images/emoji/' +
-		emojis[i] +
-		'.png" id="' +
-		emoji_names[i] +
-		'" onclick="writeEmoji(this)"/></li>';
+
+for (let i = 0; i < emojis.length; i++) {
+  const imageElement = document.createElement("img");
+  imageElement.src = `images/emoji/${emojis[i]}.png`;
+  imageElement.id = emoji_names[i];
+  imageElement.setAttribute("onclick", "javascript:writeEmoji(this)")   ;
+  emobox.appendChild(document.createElement("li").appendChild(imageElement));
```

Comment:
This seems not to be working, all `<img>` tags are there in `<ul>` without a `<li>`. Rectify this.

Reply:
Fixed this issue, Due to this existing styles are not applied. Can you review now?